hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/JdbcSqlTestSupport.java

```diff
@@ -0,0 +1,255 @@
+     */
+    protected static void assertRowsAnyOrder(String tableName, Row... rows) {
+        assertRowsAnyOrder("SELECT * FROM " + tableName, Arrays.asList(rows));
+    }
+
+    /**
+     * Assert the contents of a given table directly via JDBC
+     */
+    protected static void assertJdbcRowsAnyOrder(String tableName, Row... rows) {
+        List<Row> actualRows = jdbcRows(tableName);
+        assertThat(actualRows).containsExactlyInAnyOrderElementsOf(Arrays.asList(rows));
+    }
+
+    @Nonnull
+    protected static List<Row> jdbcRows(String tableName) {
+        List<Row> rows = new ArrayList<>();
+        try (Connection conn = DriverManager.getConnection(dbConnectionUrl);
+             Statement stmt = conn.createStatement()
+        ) {
+            stmt.execute("SELECT * FROM " + tableName);
+            ResultSet resultSet = stmt.getResultSet();
+            while (resultSet.next()) {
+                Object[] values = new Object[resultSet.getMetaData().getColumnCount()];
+                for (int i = 0; i < values.length; i++) {
+                    values[i] = resultSet.getObject(i + 1);
+                }
+                rows.add(new Row(values));
+            }
+            return rows;
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    interface TestDatabaseProvider {
+
+        String createDatabase(String dbName);
+
+        void shutdown();
+    }
+
+    static class H2DatabaseProvider implements TestDatabaseProvider {
+
+        @Override
+        public String createDatabase(String dbName) {
+            return "jdbc:h2:mem:" + dbName + ";DATABASE_TO_UPPER=false;DB_CLOSE_DELAY=-1";
+        }
+
+        @Override
+        public void shutdown() {
+            try (Connection conn = DriverManager.getConnection(dbConnectionUrl)) {
+                conn.createStatement().execute("shutdown");
+            } catch (SQLException e) {
+                throw new RuntimeException(e);
+            }
+        }
+    }
+
+    public static class PostgresDatabaseProvider implements TestDatabaseProvider {
+
+        private String jdbcUrl;
+
+        @Override
+        public String createDatabase(String dbName) {
+            jdbcUrl = "jdbc:tc:postgresql:10.21:///" + dbName + "?TC_DAEMON=true";
+            return jdbcUrl;
+        }
+
+        @Override
+        public void shutdown() {
+            if (jdbcUrl != null) {
+                ContainerDatabaseDriver.killContainer(jdbcUrl);
+                jdbcUrl = null;
+            }
+        }
+    }
+
+    public static class MySQLDatabaseProvider implements TestDatabaseProvider {
+
+        private String jdbcUrl;
+
+        @Override
+        public String createDatabase(String dbName) {
+            jdbcUrl = "jdbc:tc:mysql:8.0.29:///" + dbName + "?TC_DAEMON=true&sessionVariables=sql_mode=ANSI";
+//            jdbcUrl = "jdbc:tc:mysql:8.0.29:///" + dbName + "?TC_DAEMON=true,sessionVariables=sql_mode=''ANSI''";
```

Comment:
In last work we had a convention, that if you need to temporarily comment out some code or do some hack, then you write `// TODO HZ-1141`. With this it's easy to find all things that supposed to be fixed before release. Maybe it's worth implementing it in Hazelcast as well?

Reply:
This is a leftover. Will remove the commented line.

Reply:
I have moved the `TestDatabaseProvider` and subclasses to `hazelcast` module test jar. I need it in extensions.
7f7e89c3